modules/ROOT/pages/design-create-publish-api-specs.adoc

```diff
@@ -2,12 +2,60 @@
 
-Code editor:: Use this tool if you do have experience with writing API specifications. It offers a larger degree of freedom for creating more complex specifications. It also displays errors and warnings as problems in your specification occur.  The steps for using this tool are described in xref:design-create-publish-api-raml-editor.adoc[Create and Publish an API Specification with the Code Editor in API Designer].
+== Designing
 
-Visual editor:: Use this tool for lightweight designing and creating scaffolds. This editor does not allow you to import RAML fragments or perform several of the tasks that the code editor allows. As you fill in various forms, the editor writes your API specification for you. After creating a scaffold in this editor, you can switch to the code editor to customize your specification. The steps for using this tool are described in xref:design-create-publish-api-visual-editor.adoc[Create and Publish an API Specification with the Visual Code Editor in API Designer].
+You can use two different tools in API Designer to create API specifications in RESTful API Modeling Language (RAML) 0.8 or 1.0, or according to OpenAPI specification 2.0 (OAS).
 
-You can also use the code editor to create API fragments in RAML. The steps are described in xref:design-create-publish-api-fragment.adoc[Create and Publish an API Fragment in the Code Editor in API Designer].
+=== Code Editor
+
+With this editor, you can
+
+* Design according to best practices with a context-aware shelf that auto-populates with the appropriate methods, resources, parameters, and security components.
+* Include RAML API fragments, such as security schemas and data types, from Exchange or write your own.
+* Import existing API specifications.
+
+=== Visual Editor
+
+With this editor, you can
+
+* Create a specification for use cases in which you do not need to include RAML fragments.
+* Scaffold a specification that you can fill out and customize in the code editor.
+
+
+== Documenting
+
+As you add descriptions in your specifications and fragments, you can view them in auto-generated, interactive documentation in the right-hand pane of the code editor.
+
+== Testing
+
+You can use the mocking service to
+
+* Preview an API’s functionality before writing a single line of code
+* Send requests to the specification and receive defined responses.
+* Use behavioral headers to simulate different scenarios, such as timeouts, errors, and more.
+
+== Sharing
+
+You can publish an API specification to Anypoint Exchange for use in API Manager, Anypoint Studio, or Flow Designer. You can also add an API specification directly to API Manager.
+
+== How API Designer Relates to Other MuleSoft Products
+
+API Designer is an integral part of MuleSoft's API-led approach to integrating applications. The following diagram depicts how the API specifications that you write in API Designer can be used in other MuleSoft products.
```

Comment:
Do not make trademarks possessive; GLOBAL.
```suggestion
API Designer is an integral part of the MuleSoft API-led approach to integrating applications. The following diagram depicts how the API specifications that you write in API Designer can be used in other MuleSoft products:
```
End sentences or fragments that lead directly to a graphic, example, list, or steps with a colon.